pkgs/by-name/fi/ficsit-cli/package.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  stdenv,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "ficsit-cli";
+  version = "v0.6.0";
+  commit = "5dc8bdbaf6e8d9b1bcd2895e389d9d072d454e15";
+
+  src = fetchFromGitHub {
+    owner = "satisfactorymodding";
+    repo = "ficsit-cli";
+    rev = commit;
```

Why not use the named tag?
```suggestion
    tag = "v${version}";
```

---

Done